app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultRecordInfoPayload.kt

```diff
@@ -0,0 +1,28 @@
+package org.simple.clinic.teleconsultlog.teleconsultrecord
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import org.simple.clinic.medicalhistory.Answer
+import java.util.UUID
+
+@JsonClass(generateAdapter = true)
+data class TeleconsultRecordInfoPayload(
+    @Json(name = "recorded_at")
+    val recordedAt: String,
+
+    @Json(name = "teleconsultation_type")
+    val teleconsultationType: TeleconsultationType,
+
+    @Json(name = "patient_took_medicines")
+    val patientTookMedicines: Answer,
+
+    @Json(name = "patient_consented")
+    val patientConsented: Answer,
+
+    @Json(name = "medical_officier_number")
```

Comment:
There seems to be a typo in the JSON property name: `officier` instead of `officer`.

Since this has not gone into production and there are no consumers of this API, can we get it corrected instead of bubbling down the typo into the client code?